app/classes/car.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: float
+
+    def calculate_cost_way(self, way: float, fuel_price: float) -> float | int:
+        result = (way / 100) * self.fuel_consumption * fuel_price
+        return result
```

Comment:
can it be done in one line?

Reply:
it was in one line, but i changed logic many times and forgot exchange